extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -923,6 +925,10 @@ export class AmpA4A extends AMP.BaseElement {
         return;
       }
       return this.mutateElement(() => {
+        // Fire an ad-refresh event so that 3rd parties can track when an ad
+        // has changed.
+        this.maybeTriggerAnalyticsEvent_('adRefresh');
```

Comment:
Copying same question from previous PR: Do we need any extra variables here, or are the vars introduced by `maybeTriggerAnalyticsEvent_` sufficient?

Reply:
or should you just call `triggerAnalyticsEvent` here directly? Are those `analyticsVars` really relevant here?

Reply:
I wasn't sure the exact use case. If you think it is unnecessary I can definitely call `triggerAnalyticsEvent` directly.

Reply:
Done.